builds/e2e/templates/nested-longhaul-deploy.yaml

```diff
@@ -23,8 +23,6 @@ parameters:
   logAnalytics.sharedKey: ''
   logAnalytics.logType: ''
   twinTester.twinUpdateSize: ''
-  twinTester.twinUpdateFailureThreshold: ''
-  twinTester.edgeHubRestartFailureTolerance: ''
```

Comment:
We were never even using these values.

Reply:
Was it you who made these param changes a while ago?

Reply:
No. These param headers were copied from the original longhaul deployment without being integrated in:

https://github.com/Azure/iotedge/blame/39d324910d76bc6ebe62a90e9d4ddc77f8b53be3/builds/e2e/templates/nested-longhaul-deploy.yaml#L26

https://github.com/Azure/iotedge/pull/4333/files#diff-3e30d323ba6fc94aa3edf4e4cf11b37ffafbc81bfb4c02c8fcddb0759ab183d7R27